dotnet/samples/01.messaging.echoBot/Properties/launchSettings.json

```diff
@@ -0,0 +1,25 @@
+{
+	"profiles": {
+		"Microsoft Teams (browser)": {
+			"commandName": "Project",
+			"launchBrowser": true,
+			"launchUrl": "https://teams.microsoft.com/l/app/d8053def-e1a6-45b1-9650-8142a91341c2?installAppPackage=true&webjoin=true&appTenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&login_hint=kavinsingh@microsoft.com",
```

Comment:
Leave this as a place holder, e.g., https://teams.microsoft.com/.
The app ID, app tenant ID, and login hint will be filled by Teams Toolkit automatically when "prepare teams app dependencies".

Reply:
resolved